qa/zcash/full_test_suite.py

```diff
@@ -87,24 +93,40 @@ def test_fortify_source(filename):
 def check_security_hardening():
     ret = True
 
-    # PIE, RELRO, Canary, and NX are tested by make check-security.
-    ret &= subprocess.call(['make', '-C', repofile('src'), 'check-security']) == 0
+    Makefile_file_path = './src/Makefile'
+
+    if os.path.exists(Makefile_file_path):
+        ret &= subprocess.call(['make', '-C', repofile('src'), 'check-security']) == 0
+    else:
+        # Equivalent to make check-security (this is just for CI purpose)
+        bin_programs = ['./src/zcashd', './src/zcash-cli', './src/zcash-tx', './src/bench/bench_bitcoin']  # Replace with actual values
+        bin_scripts = ['./src/zcash-inspect', './src/zcashd-wallet-tool']   # Replace with actual values
+
+        print(f"Checking binary security of {bin_programs + bin_scripts}...")
+
+        for program in bin_programs:
+            command = ['./contrib/devtools/security-check.py', program]
+            ret &= subprocess.call(command) == 0
+
+        for script in bin_scripts:
+            command = ['./contrib/devtools/security-check.py', '--allow-no-canary', script]
+            ret &= subprocess.call(command) == 0
 
     # The remaining checks are only for ELF binaries
     # Assume that if zcashd is an ELF binary, they all are
-    with open(repofile('src/zcashd'), 'rb') as f:
+    with open('./src/zcashd', 'rb') as f:
```

Comment:
Why was this changed to not use `repofile`? `full_test_suite.py` uses that to make itself independent of the directory in which it is run. The other new path additions should be using it as well.

Reply:
Good observation, it's better to use `repofile`. I think I accidentally removed that while running a unit test. I'll revert that in the next commit